.travis.yml

```diff
@@ -4,6 +4,10 @@ services:
 env:
   global:
     - IMAGE=qgis/qgis
+    - GIT_NAME: "'Automatic Publish'"
+    - GIT_EMAIL: jan.caha@outlook.org
```

Comment:
Wouldn't these be better set on Travis itself?

Reply:
Moved to travis config. It is probably better there.